packages/@okta/vuepress-site/docs/guides/sign-into-spa/use-the-access-token/react/getaccesstoken.md

```diff
@@ -1,7 +1,7 @@
-Get the access token using the [getAccessToken()](https://github.com/okta/okta-oidc-js/tree/master/packages/okta-react#authgetaccesstoken) method on the [Auth](https://github.com/okta/okta-oidc-js/tree/master/packages/okta-react#auth) object. Then, use this value to add an `Authorization` header to outgoing requests:
+Get the access token using the [accessToken](https://github.com/okta/okta-oidc-js/tree/master/packages/okta-react#authstateaccesstoken) property on the [AuthState](https://github.com/okta/okta-oidc-js/tree/master/packages/okta-react#authstate) object. Then, use this value to add an `Authorization` header to outgoing requests:
```

Comment:
I don't think the `AuthState.accessToken` do section exists as a heading you can link to.